features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/OtelTracerProvider.kt

```diff
@@ -245,6 +306,14 @@ class OtelTracerProvider(
         internal const val DEFAULT_SERVICE_NAME_IS_MISSING_ERROR_MESSAGE =
             "Default service name is missing during" +
                 " OtelTracerProvider creation, did you initialize SDK?"
+        internal const val RUM_NOT_ENABLED_ERROR_MESSAGE =
+            "You're trying to bundle the traces with a RUM context, " +
+                "but the RUM feature was disabled in your Configuration. " +
```

Comment:
we need to update this message (and similar message in `AndroidTracer`) - RUM is not enabled in core `Configuration` class anymore in v2.